src/lib.rs

```diff
@@ -39,6 +39,10 @@ impl str::FromStr for Language {
     }
 }
 
+pub fn parse_spec_str_or_panic(src: &str) -> ast::Spec {
+    parser::parse(src)
```

Comment:
It's a bit embarassing that the parser is so dirty at the moment. Maybe it needs a good helping `anyhow`, but I guess that's a separate issue?

Reply:
force-pushed over this